PyQt5-stubs/QtCore.pyi

```diff
@@ -5994,6 +5994,10 @@ class QPoint(sip.simplewrapper):
 
     def __neg__(self) -> 'QPoint': ...
     def __pos__(self) -> 'QPoint': ...
+    def __add__(self, point: 'QPoint') -> 'QPoint': ...
+    def __sub__(self, point: 'QPoint') -> 'QPoint': ...
+    def __mul__(self, point: 'QPoint') -> 'QPoint': ...
+    def __truediv__(self, point: 'QPoint') -> 'QPoint': ...
```

Comment:
Why `__truediv__` rather than `__div__`? Also, same as above.

Reply:
I think use Python 3 operator (`__truediv__` and `__floordiv__`) will be better than Python 2 (`__truediv__` and `__div__`).

Reply:
Ah, true, I read the docs wrong there. My bad!